src/RangedatePicker.vue

```diff
@@ -18,7 +18,7 @@
             <ul v-for="r in 6" :class="[s.days]" :key="r">
               <li :class="[{[s.daysSelected]: isDateSelected(r, i, 'first', startMonthDay, endMonthDate),
               [s.daysInRange]: isDateInRange(r, i, 'first', startMonthDay, endMonthDate),
-              [s.dateDisabled]: isDateDisabled(r, i, startMonthDay, endMonthDate)}]" v-for="i in numOfDays" :key="i" v-html="getDayCell(r, i, startMonthDay, endMonthDate)"
+              [s.dateDisabled]: isDateDisabled(r, i, startMonthDay, endMonthDate), [s.dateAfterMax]: isDateAfterMax(r, i, 'first', startNextMonthDay, endNextMonthDate)}]" v-for="i in numOfDays" :key="i" v-html="getDayCell(r, i, startMonthDay, endMonthDate)"
```

Comment:
@AaronThornton00 thank you for the improvements :+1: 
I noticed there is a bug here, it should be `isDateAfterMax(r, i, 'first', startMonthDay, endMonthDate)`